rules/opentitan.bzl

```diff
@@ -802,3 +802,46 @@ def opentitan_functest(
         name = name,
         tests = all_tests,
     )
+
+def _get_or_empty(dict, key):
```

Comment:
Can you check the type of dict so you don't get passed a list of deps and ignore it all quietly here?